frontend/taipy-gui/src/components/icons/MatchCase.tsx

```diff
@@ -0,0 +1,9 @@
+import { SvgIcon, SvgIconProps } from "@mui/material";
```

Comment:
`import SvgIcon, { SvgIconProps } from "@mui/material/SvgIcon";` for better tree shaking